src/components/userbuttons.jsx

```diff
@@ -0,0 +1,43 @@
+import DefaultBtn from './defaultbtn';
+import DeleteBtn from './deletebtn';
+import ResetBtn from './resetbtn';
+import EqualBtn from './equaltobtn';
+
+const UserButtons = () => {
+  return (
+    <div class="user-butns h-[65vh]">
+      <div class="calculator-btns mt-5 w-[300px] rounded-lg bg-themeOne-mobile-button-background p-5 ">
+        <div class="row  mb-2 flex items-center justify-between">
+          <DefaultBtn value="7" />
+          <DefaultBtn value="8" />
+          <DefaultBtn value="9" />
+          <DeleteBtn />
```

Comment:
All button should be `<Button />` component, what you would use to differentiate them is their props.